clients/routes.ts

```diff
@@ -1,4 +1,7 @@
 const routes = {
+  apiEntreprise: {
+    association: '/v4/ministere_interieur/api-association/associations/',
```

Comment:
Petite update ici: on a "changé" l'url (😅) maintenant faut utiliser ma suggestion.

On a demander aux quelques personnes qui ont déjà implem de changer l'url btw.

```suggestion
    association: '/v4/djepva/api-association/associations/',
```